src/test/java/chips/ChipsClientTest.java

```diff
@@ -0,0 +1,33 @@
+package chips;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.companieshouse.api.strikeoffobjections.chips.ChipsClient;
+import uk.gov.companieshouse.api.strikeoffobjections.common.ApiLogger;
+import uk.gov.companieshouse.api.strikeoffobjections.groups.Unit;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@Unit
+@ExtendWith(MockitoExtension.class)
+class ChipsClientTest {
+
+    private static final String REQUEST_ID = "test123";
+
+    @InjectMocks
+    private ChipsClient chipsClient;
+
+    @Mock
+    private ApiLogger apiLogger;
+
+    @Test
+    void testSendMessageToChips(){
+        // TODO OBJ-240 test sending contact data to CHIPS
+        chipsClient.sendToChips(REQUEST_ID);
+        assertEquals("test123", REQUEST_ID);
```

Comment:
This assertion is pretty pointless - it's checking the unit-test code not the implementation. Appreciate there's very little at the moment in the implementation to check. Maybe this test class just isn't required at this point?

Reply:
the real method will call apiLogger, so you could verify that apiLogger is called...

Reply:
Made assertion more meaningful - please bear mind that this is a stub.